docs/01_overview/01_framework_and_toolchain.md

```diff
@@ -4,10 +4,10 @@ content_title: Framework and Toolchain
 
 The Antelope framework is made up of the following components and toolchain:
 
-1. `nodeos` (node + Antelope = nodeos):  the core Antelope node daemon that can be configured with plugins to run a node. Example uses are block production, dedicated API endpoints, and local development.
-2. `cleos` (CLI + Antelope = cleos): the command line interface to interact with the blockchain and to manage wallets.
-3. `keosd` (key + Antelope = keosd): the component that securely stores Antelope keys in wallets.
-4. `CDT`: toolchain for WebAssembly (Wasm)  and a set of tools to facilitate smart contract writing for the Antelope framework.
+1. `nodeos`:  the core Antelope node daemon that can be configured with plugins to run a node. Example uses are block production, dedicated API endpoints, and local development.
+2. `cleos`: the command line interface to interact with the blockchain and to manage wallets.
+3. `keosd`: the component that securely stores Antelope keys in wallets.
+4. `CDT`: the Contract Development Toolkit is a C/C++ toolchain targeting WebAssembly (WASM) and a set of tools to facilitate development of smart contracts written in C/C++ that are meant to be deployed to an Antelope blockchain..
```

Comment:
Suggestion: simplify this long sentence to "...facilitate development of smart contracts written in C/C++ for Antelope-based blockchains."

Also, remove the double period at the end (..)